js/drop.js

```diff
@@ -36,6 +36,12 @@ export const DropAPI = superclass =>
             return productDrop;
         }
 
+        async createProductDropAndQueryDrop(id, payload) {

```

Comment:
Remove the "and" to keep it as dumb as possible and type it on types/drop.d.ts

```suggestion
        async createProductDropQueryDrop(id, payload) {
```